engine/src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -239,12 +239,36 @@ object WorkflowDescriptor {
   val CallLogsDirOptionKey = "call_logs_dir"
   val OptionKeys: Set[String] = Set(WorkflowLogDirOptionKey, WorkflowOutputsOptionKey, CallLogsDirOptionKey)
 
+  // This is, frankly, an aberration. But don't worry - it's going away soon :D
+  // TODO: KILL IT WITH FIRE!
+  def getBackendFromOptions(optionsString: String): Backend = {
+    val backendString = WorkflowOptions.fromJsonString(optionsString) flatMap { x => x.get("backend")}
+    backendString match {
+      case Success(x) => CromwellBackend.backend(x)
+      case Failure(_) => CromwellBackend.defaultBackend
+    }
+  }
+
+  /**
+    * Warning: Uses the default backend for this workflow.
+    */
   def apply(id: WorkflowId, sourceFiles: WorkflowSourceFiles): WorkflowDescriptor = {
```

Comment:
Thanks @gauravs90! Yeah, I didn't *want* these constructors, they're no-longer current. But I didn't want to re-plumb everything else to work with my new temporary constructor which I'm pretty sure won't last very long either (apparently, it won't last past the end of your PR!)